src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java

```diff
@@ -61,10 +62,14 @@
   private String exactRemaining;
 
   /**
+   * The issue closed.
+   */
+  private boolean isClosed = false;
```

Comment:
isClosed is the name of the getter normally. The member variable is simply *closed"